### DISCUSSION TITLE
The project cannot run locally

### DISCUSSION THREAD

**Discussion by Tzyito** [toeverything/AFFiNE#6223] [Q&A]
When I run `yarn install`, I get the following error. Is this a problem with my yarn version? This problem did not occur when installing other projects
<img width="1009" alt="image" src="https://github.com/toeverything/AFFiNE/assets/37643993/24806e74-0187-430b-aa14-bccce8b4d0d6">

**Comment by EYHN** [comment]
you need to check this file

![CleanShot 2024-03-21 at 01 11 40@2x](https://github.com/toeverything/AFFiNE/assets/13579374/18d6705b-fff0-4c8d-9e57-3d520053638a)

**Reply by Tzyito** [reply]
@EYHN 
Hi, Thanks for your reply. In addition to this file, the error message in my terminal is: **** must be built because it never has been before or the last one failed. Is this the normal information for installation?

build.log
```
# This file contains the result of Yarn building a package (@affine/server@workspace:packages/backend/server)
# Script name: postinstall

Error: request to https://binaries.prisma.sh/all_commits/efd2449663b3d73d637ea1fd226bafbcf45b3102/darwin-arm64/libquery_engine.dylib.node.gz.sha256 failed, reason: 
```

**Reply by Tzyito** [reply]
btw, my node version is 4.1.0, yarn version is 20.10.0

**Reply by Brooooooklyn** [reply]
@Tzyito It seems like there was a problem while downloading the prisma binary according to the log. You can try reinstalling it a few more times.
